scripts/run_integration_tests.sh

```diff
@@ -29,6 +36,13 @@ if [ ! -f "$CLEAR_OS_RAW_IMAGE" ]; then
     pushd $WORKLOADS_DIR
     time qemu-img convert -p -f qcow2 -O raw $CLEAR_OS_IMAGE_NAME $CLEAR_OS_RAW_IMAGE_NAME
     popd
+else
+    CLEAR_OS_RAW_IMAGE_SHA1SUM_REF="2be37c503cae9994f9b061d67a321b6241a730e5"
+    CLEAR_OS_RAW_IMAGE_SHA1SUM=$(sha1sum $CLEAR_OS_RAW_IMAGE | cut -d ' ' -f 1)
+    if [ "$CLEAR_OS_RAW_IMAGE_SHA1SUM" != "$CLEAR_OS_RAW_IMAGE_SHA1SUM_REF" ]; then
+        echo "ERROR: $CLEAR_OS_RAW_IMAGE has the wrong SHA1SUM $CLEAR_OS_RAW_IMAGE_SHA1SUM (expecting $CLEAR_OS_RAW_IMAGE_SHA1SUM_REF)\nConsider removing $CLEAR_OS_RAW_IMAGE to fix the issue."
+        exit 1
```

Comment:
I'd rather you used `sha1sum --check` and checked them all in one go? Maybe from a file in the bucket?

i.e  `curl --silent "https://cloudhypervisorstorage.blob.core.windows.net/images/sha1sums" | sha1sum --check`

Reply:
Oh yeah sounds good I'm gonna do that :)